optimade/client/client.py

```diff
@@ -121,7 +121,9 @@ def __init__(
             max_results_per_provider: The maximum number of results to download
                 from each provider.
             headers: Any additional HTTP headers to use for the queries.
-            http_timeout: The HTTP timeout to use per request.
+            http_timeout: The timeout to use per request. Defaults to 10
+                seconds with 60 seconds for reads specifically. Overriding this value
```

Comment:
If I understand it correctly, you defined the read timeout on line 79 as 1000 seconds, yet in this docstring you say it is 60 seconds.
```suggestion
                seconds with 1000 seconds for reads specifically. Overriding this value
```